crates/autopilot/src/database/competition.rs

```diff
@@ -17,19 +17,19 @@ use {
 
 #[derive(Clone, Debug)]
 pub enum ExecutedFee {
-    /// Fee is calculated by the solver and known upfront (before the settlement
+    /// Fee is determined by user and known upfront (before the settlement
     /// is finalized).
-    Solver(U256),
+    UserFee(U256),
```

Comment:
nit: it's not really UserDetermined, but rather OrderDetermined? 

How about `ExecutedFee:Signed` vs `ExecutedFee::Surplus` ?